app/components/SourceSelector.vue

```diff
@@ -18,27 +18,61 @@
         @click="sourceProperties"/>
     </div>
   </div>
-  <selector
-    class="studio-controls-selector"
-    @contextmenu="showContextMenu"
-    @dblclick="sourceProperties"
-    :items="sources"
-    :activeItems="activeItemIds"
-    @select="makeActive"
-    @sort="handleSort">
-    <template slot="actions" slot-scope="props">
-      <i
-        class="fa fa-lock icon-btn source-selector-action"
-        :class="lockClassesForSource(props.item.value)"
-        @click.stop="toggleLock(props.item.value)"
-        @dblclick.stop="() => {}" />
-      <i
-        class="fa fa-eye icon-btn source-selector-action"
-        :class="visibilityClassesForSource(props.item.value)"
-        @click.stop="toggleVisibility(props.item.value)"
-        @dblclick.stop="() => {}" />
-    </template>
-  </selector>
+
+    <sl-vue-tree
+        :value="nodes"
+        @select="makeActive"
+        @drop="handleSort"
+        @toggle="toggleFolder"
```

Comment:
Looks like these are being done with 4 space indents.  I'm not sure if your IDE is doing that, but I think we've generally been doing normal 2 space indents for this situation.